### PR TITLE
add notebook for add/remove ssh keys to slivers

### PR DESCRIPTION
**Pull request by kthare10**
add notebook for add/remove ssh keys to slivers

### CODE REVIEW

File: fabric_examples/fablib_api/ssh_keys/ssh_keys.ipynb [hunk 1]
```diff
@@ -0,0 +1,291 @@
+{
```
Comment (sajith) [inline fabric_examples/fablib_api/ssh_keys/ssh_keys.ipynb:1]: <p>It doesn't seem that <code>nic_name</code> and <code>model_name</code> declared here are not used later. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fabric-testbed/jupyter-examples/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (sajith) [inline fabric_examples/fablib_api/ssh_keys/ssh_keys.ipynb:1]: <p>This block does not seem to be used at all. Do we need it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fabric-testbed/jupyter-examples/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (sajith) [inline fabric_examples/fablib_api/ssh_keys/ssh_keys.ipynb:1]: <p><code>key.write_private_key_file(filename=private_key_file)</code>  would be shorter.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fabric-testbed/jupyter-examples/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

**Review by sajith — APPROVED** [review]
I left some comments via ReviewNB.  This is my first time using it to leave some comments, and the experience of writing them on their site was just fine.  On GitHub those comments kind of looks odd. Hopefully this still works for everyone.

File: fabric_examples/fablib_api/ssh_keys/ssh_keys.ipynb (hunk 1 above)
Comment (kthare10) [reply fabric_examples/fablib_api/ssh_keys/ssh_keys.ipynb:1]: <p>subnet is passed to the slice.add_l2network in create slice block.</p><p> </p>

**Pull request merged**